### PR TITLE
Simplify manpage, add .gitignore

### PR DESCRIPTION
**Pull request by The-King-of-Toasters**

### CODE REVIEW

**Comment by eradman** [comment]
The suggestion to use .gitignore is reasonable, but it isn't the habit on any of my personal projects. I tend to run `make clean` then `git status` to see if the `clean:` make rule missed anything

File: entr.1 [hunk 1]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
 is replaced with the absolute path of the first file to trigger an event.
 The first file under watch is used as the default.
 If the
 .Fl s
-option is used, the name of the first file to triggler an event can be read from
+option is used, the name of the first file to trigger an event can be read from
 .Va $0 .
-.Sh COMMANDS
+.Ss Commands
 .Nm
```
Comment (eradman) [inline entr.1]: Interesting observation, but I'll keep the pattern that https://man.openbsd.org/less uses

File: entr.1 [hunk 2]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
 listens for keyboard input and responds to the following commands:
-.Bl -tag -width Ic
-.It SPACE
-Execute the utility immediately.
+.Bl -tag -width <space>
```
Comment (eradman) [inline entr.1]: I get it; yes thanks

File: entr.1 [hunk 3]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
 .Fl r
 option is set this will terminate and restart the child process as if a file
 change event had occurred.
-.It q
-Quit; equivalent pressing Ctrl\-C.
+.It Cm q
+Quit
+.Nm .
+Equivalent to pressing
+.Aq Cm control-c
```
Comment (eradman) [inline entr.1]: There is a wide style differences in existing man pages on this point--other man pages use every possible combination. Probably keep as is for now (although the `\` is not needed is it)

File: entr.1 [hunk 4]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
+or sending
+.Dv SIGINT .
```
Comment (eradman) [inline entr.1]: True, but I think this is implied by Ctrl-C

File: entr.1 [hunk 5]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
 .Sh ENVIRONMENT
-If
-.Ev PAGER
-is undefined,
+.Bl -tag -width PAGER
+.It Ev PAGER
+If undefined,
 .Nm
-will assign
-.Pa /bin/cat
```
Comment (eradman) [inline entr.1]: Again, this exact path is used

File: entr.1 [hunk 6]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
+assigns it to
+.Xr sh 1 .
+.El
 .Sh EXIT STATUS
 The
 .Nm
 utility exits with one of the following values:
 .Pp
-.Bl -tag -width Ds -offset indent -compact
-.It Li 0
```
Comment (eradman) [inline entr.1]: ok suggestion, but I'm going to keep it the same as https://man.openbsd.org/diff

File: entr.1 [hunk 7]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
+.It 1
+No regular files were provided as input or an error occurred.
+.It 2
+A file was added to a directory and the directory watch option was specified.
 .El
 .Sh EXAMPLES
 Rebuild a project if source files change, limiting output to the first 20 lines:
 .Pp
-.Dl $ find src/ | entr -s 'make | head -n 20'
+.Dl $ find src/ | entr -s 'make | sed 20q'
```
Comment (eradman) [inline entr.1]: That's cute. We'll go with this

File: entr.1 [hunk 8]
```diff
@@ -165,3 +172,7 @@ Clear the screen and run a query after the SQL script is updated:
 Rebuild project if a source file is modified or added to the src/ directory:
 .Pp
 .Dl $ while true; do ls src/*.rb | entr -d make; done
+.Sh SEE ALSO
+.Xr cat 1 ,
+.Xr clear 1 ,
+.Xr sh 1
```
Comment (eradman) [inline entr.1]: Not a bad suggestion, but I think not needed

**Comment by wavexx** [comment]
On Mon, Nov 18 2019, Eric Radman wrote:
> The suggestion to use .gitignore is reasonable, but it isn't the habit
> on any of my personal projects. I tend to run `make clean` then `git
> status` to see if the `clean:` make rule missed anything

I find .gitignore a useful tool to avoid committing object files from a
working tree.

`git ls-files -o` will still tell you what you need after that.

**Comment by eradman** [comment]
@wavexx you're right that .gitignore prevents adding object files. That's a good reason...

File: entr.1 [hunk 9]
```diff
@@ -37,35 +41,30 @@ interactive utilities.
 The arguments are as follows:
 .Bl -tag -width Ds
 .It Fl a
-Respond to all events which occur while the
```
[the remark this answers is not in the document]
Comment (The-King-of-Toasters) [reply entr.1]: The manual already stated that `utility` gets executed, so there's no need to refer to it as *the* utility. This is just a personal preference however.

File: entr.1 [hunk 10]
```diff
@@ -37,35 +41,30 @@ interactive utilities.
...
 is running.
-Without this option
+Without this option,
 .Nm
 consolidates events in order to avoid looping.
-This option has no effect in conjunction with the
-.Fl r
-flag.
+Has no effect when used with
+.Fl r .
```
[the remark this answers is not in the document]
Comment (The-King-of-Toasters) [reply entr.1]: Disagree, users are smart enough to know what a flag looks like and to search for what `-r` does.

File: entr.1 [hunk 11]
```diff
@@ -37,35 +41,30 @@ interactive utilities.
...
 .It Fl c
-Execute
-.Pa /usr/bin/clear
-before invoking the
-.Ar utility
-specified on the command line.
+Run
+.Xr clear 1
```
[the remark this answers is not in the document]
Comment (The-King-of-Toasters) [reply entr.1]: Agreed, I'll revert these types of changes.

File: entr.1 [hunk 12]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
-Control of the TTY is not transferred the child process.
+Control of the TTY is not transferred to the child process.
 .It Fl s
-Evaluate the first argument using the interpreter specified by the
-.Ev SHELL
-environment variable.
-If stdout is a TTY, the name of the shell and exit code is printed after each
-invocation.
+Evaluate the first argument using the interpreter specified by
+.Ev SHELL .
```
[the remark this answers is not in the document]
Comment (The-King-of-Toasters) [reply entr.1]: I guess this comes down to personal preference. `SHELL` comes up in so many manuals that saying it's an environment variable it stating the obvious.

File: entr.1 [hunk 13]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
+If standard output is a TTY, the name of the interpreter and exit code is
```
[the remark this answers is not in the document]
Comment (The-King-of-Toasters) [reply entr.1]: Please don't. `stdout` is a programmer shorthand for "Standard Output" that is used in all official UNIX/POSIX documentation.

File: entr.1 (hunk 1 above)
Comment (The-King-of-Toasters) [reply entr.1]: I think it's cleaner this way, as the command list is so short there's no benefit to splitting it out into another section.

File: entr.1 (hunk 3 above)
Comment (The-King-of-Toasters) [reply entr.1]: I'm basing this change on how `vi(1)` looks.

File: entr.1 (hunk 4 above)
Comment (The-King-of-Toasters) [reply entr.1]: Better to be explicit in this case I think.

**Comment by The-King-of-Toasters** [comment]
Email is preferred: `Stephen Gregoratto <dev@sgregoratto.me>`

File: entr.1 (hunk 13 above)
[the remark this answers is not in the document]
Comment (eradman) [reply entr.1]: I see your point, and manuals like grep(1) and echo(1) seem to agree

File: entr.1 (hunk 1 above)
Comment (eradman) [reply entr.1]: You're right that it is short... I'll look again

File: entr.1 (hunk 3 above)
Comment (eradman) [reply entr.1]: vi(1) needs to be pedantic about it's description of keyboard input, so this is a good example

File: entr.1 [hunk 14]
```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
...
+.Bl -tag -width 2n -offset indent -compact
+.It 0
 .Dv SIGINT
 or
 .Dv SIGTERM
-was received
-.It Li 1
-No regular files were provided as input or an error occurred
-.It Li 2
-A file was added to a directory and the directory watch option was specified
```
Comment (eradman) [inline entr.1]: Good observation; `Li` is not needed

**Comment by eradman** [comment]
Pushed a subset of these changes. Appreciate the the careful reading @The-King-of-Toasters

**Pull request closed without merging**